src/main/java/slave/Deadline.java

```diff
@@ -0,0 +1,56 @@
+package slave;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * a Deadline object is a Task with a deadline
+ */
+public class Deadline extends Task {
+    private LocalDate deadline;
+
+    /**
+     * Creates a Deadline object with the specified parameters
+     *
+     * @param taskName is the task name
+     * @param deadline is the deadline of the task
+     */
+    public Deadline(String taskName, LocalDate deadline) {
+        super(taskName);
+        this.deadline = deadline;
+    }
+
+    /**
+     * Creates a Deadline object with the specified parameters
+     *
+     * @param isCompleted is the state of completeness fo the task
+     * @param taskName    is the task name
+     * @param deadline    is the deadline of the task
+     */
+    protected Deadline(boolean isCompleted, String taskName, LocalDate deadline) {
+        super(isCompleted, taskName);
+        this.deadline = deadline;
+    }
+
+    public String getDeadline() {
+        return this.deadline.format(DateTimeFormatter.ofPattern("d MMM yyyy"));
+    }
+
+    public LocalDate getRawDeadline() {
+        return this.deadline;
+    }
```

Comment:
A method comment will be useful to distinguish between these two

So getDeadline can be getFormattedDeadline